### PR TITLE
New Portal Cheat Sheet

### PR DESCRIPTION
**Pull request by tagawa**
https://duck.co/ia/view/portal_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=portal cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/portal_cheat_sheet_mobile.png)

**Comment by tagawa** [comment]
Here's a strange thing - the square brackets are missing off the `[Tab]` key in the IA, but are there in the JSON file. Anyone know why?

portal-cheatsheet

File: share/goodie/cheat_sheets/json/portal.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "section_order": [
+        "Movement",
+        "Actions",
+        "Co-operation",
+        "Miscellaneous"
+    ],
+    "sections": {
+        "Movement": [{
+            "val": "Move forward",
+            "key": "[W]"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: @tagawa For single key presses we shouldn't be wrapping the keys with brackets, see [the docs](http://docs.duckduckhack.com/frontend-reference/cheat-sheet-reference.html#key-press-style-suggestions).

**Comment by tagawa** [comment]
Agh, I wasn't thinking. Thanks @GuiltyDolphin.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=portal cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/portal_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/portal.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
+{
+    "id": "portal_cheat_sheet",
+    "name": "Portal",
+    "description": "Default controls for the PC version of Portal 1 & 2. Keys are user-definable in Options, via the game's Developer Console.",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: I think just `Default key bindings for the PC version of Portal and Portal 2` would be fine.

File: share/goodie/cheat_sheets/json/portal.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            "val": "Move left (strafe)",
+            "key": "S"
+        }, {
+            "val": "Move right (strafe)",
+            "key": "D"
+        }, {
+            "val": "Jump",
+            "key": "Space"
+        }, {
+            "val": "Duck (crouch) ",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: Rogue space at the end here.

File: share/goodie/cheat_sheets/json/portal.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            "key": "Alt"
+        }],
+        "Actions": [{
+            "val": "Fire blue portal",
+            "key": "Left mouse button"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: We could try `[Left mouse]` or `[LMB]`.

File: share/goodie/cheat_sheets/json/portal.json [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        }, {
+            "val": "Fire orange portal",
+            "key": "Right mouse button"
+        }, {
+            "val": "Use item (buttons, machines, etc.)",
+            "key": "E"
+        }, {
+            "val": "Toggle zoom",
+            "key": "Middle mouse button"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: So... `[Middle mouse]` maybe?

File: share/goodie/cheat_sheets/json/portal.json [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+        }, {
+            "val": "Zoom in",
+            "key": "Scroll wheel up"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: How about `[Mouse wheel \\{up\\}]`?

File: share/goodie/cheat_sheets/json/portal.json (hunk 2 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/portal.json]: Good idea. I'll keep the term "controls" as it sounds less technical.

File: share/goodie/cheat_sheets/json/portal.json (hunk 3 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/portal.json]: Thanks

File: share/goodie/cheat_sheets/json/portal.json (hunk 6 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/portal.json]: That sounds better

File: share/goodie/cheat_sheets/json/portal.json (hunk 4 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/portal.json]: I'd prefer to leave `button` there for clarity

**Comment by tagawa** [comment]
Thanks for the further comments @GuiltyDolphin

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=portal cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/portal_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/portal.json [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    "metadata": {
+        "sourceName": "Portal Unofficial Wiki",
+        "sourceUrl": "http://theportalwiki.com/wiki/List_of_default_keys"
+    },
+    "aliases": ["portal 2", "portal2"],
+    "template_type": "keyboard",
+    "section_order": [
+        "Movement",
+        "Actions",
+        "Co-operation",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/portal.json]: Maybe just `Co-op` or `Co-op Mode`.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=portal cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/portal_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/portal.json (hunk 7 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/portal.json]: `Co-op Mode` sounds good - thanks.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=portal cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/portal_cheat_sheet_mobile.png)

**Comment by GuiltyDolphin** [comment]
@tagawa Okay, cool - this looks great! :dolphin:

**Comment by daxtheduck** [comment]
@tagawa this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by tagawa** [comment]
Much obliged Mr. Dolphin

**Pull request merged**